matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/session/room/create/CreateRoomTask.kt

```diff
@@ -54,6 +54,7 @@ internal class DefaultCreateRoomTask(private val roomAPI: RoomAPI,
             handlerThread.start()
             val handler = Handler(handlerThread.looper)
 
+            // TODO Maybe do the same code for join room request ?
```

Comment:
Maybe use RealmQueryLatch ? Should be tested first

Reply:
RealmQueryLatch is not used for the moment, why?